__tests__/app.test.js

```diff
@@ -175,6 +175,25 @@ describe("Articles endpoint", () => {
           expect(body.text).toBe("Bad Request");
         });
     });
+    test("GET:200 - Returns all articles filtered by specified topic", () => {
+      return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+        .then((body) => {
+          body.body.articles.forEach((topic) => {
```

Comment:
variable naming could be better.  What you've labelled body is actually your response.  you should just de-structure your body from result.  You are also iterating over your articles, not topics.  so name the variable in your for each accordingly.